### DISCUSSION TITLE
Adding INP as a core web vital to the Playwright Engine

### DISCUSSION THREAD

**Discussion by joshuscurtis** [artilleryio/artillery#2454] [Artillery]
Would it be possible to add INP as a metric to be recorded and measured as part of the Playwright Engine?
I think the code to measure is already part of the Core Web Vitals Library - so it would just be a case of outputting this as a metric.  

Is this something the is on the horizon, as Google will start using this metric from March?

**Comment by hassy** [comment]
hi @joshuscurtis 👋 we're adding `INP` in the next release of Artillery

**Reply by joshuscurtis** [reply]
Amazing - thank you @hassy

**Comment by bernardobridge** [comment]
👋 INP metrics will be automatically reported (when available) in the new version: [Artillery v2.0.5](https://github.com/artilleryio/artillery/releases/tag/artillery-2.0.5)

**Reply by joshuscurtis** [reply]
Rapid! Amazing thanks for letting me know - will try it out now!
